assets/constants.ts

```diff
@@ -5302,5 +5302,12 @@ export = {
   // Maximum allowed length of a username.
   "MAX_USERNAME_LENGTH": 30,
 
+  // Docstring section headers split up into arguments, returns, yields and
+  // and raises sections
+  "DOCSTRING_SECTION_ARGS": "args",
```

Comment:
This shouldn't be here. constants.ts is for constants that are shared between the frontend and backend, which this one isn't. 

Put the constants at the top of the file it refers to, or within the class it refers to -- scope them as narrowly as possible.